adapters/sequelize.js

```diff
@@ -103,9 +104,18 @@ module.exports = function (model, opts) {
 
   return P.all([columns, associations])
     .then(function () {
+      var isCompositePrimary = false;
+      var idField = _.keys(model.primaryKeys)[0];
+
+      if (_.keys(model.primaryKeys).length > 1) {
+        isCompositePrimary = true;
+        idField = 'forestCompositePrimary';
+      }
+
       return {
         name: model.name,
-        idField: _.keys(model.primaryKeys)[0],
+        idField: idField,
+        isCompositePrimary: isCompositePrimary,
         fields: fields
```

Comment:
I think that theses changes are not related to the field type support. Don't you think?